src/main/java/net/dv8tion/jda/api/interactions/callbacks/IReplyCallback.java

```diff
@@ -87,7 +87,6 @@ public interface IReplyCallback extends IDeferrableCallback
      * <br>When a message is ephemeral, it will only be visible to the user that used the interaction.
      * <br>Limitations:
      * <ul>
-     *     <li>Cannot be deleted by the bot</li>
```

Comment:
This needs to be removed in `InteractionHook`, `ReplyCallbackAction` and `WebhookMessageCreateAction`